bindings/python/multibody/frame.hpp

```diff
@@ -51,12 +51,24 @@ namespace pinocchio
                           "A Plucker coordinate frame related to a parent joint inside a kinematic tree.\n\n",
                           bp::no_init
                          )
+        .def(bp::self == bp::self)
```

Comment:
Please move this line in the dedicated visit section of the Visitor.
Could you also add `.def(bp::self != bp::self)`

Reply:
I think I had not put it because there was no direct correspondent in C++, but I'll see... Anyhow, what's the point of adding `!=` explicitly? It is automatically added if `==` exists, the unit tests check that

Reply:
There is no reason for `!=` to be added as far as I know. Indeed, some class may not have it.

Reply:
Sorry, I think it wasn't clear, what I meant was, why adding `.def(bp::self != bp::self)` explicitly? I am testing `!=` in the Python unit tests and it already works without it, which makes me think it must be defined automatically from `==`

Reply:
> There is no reason for `!=` to be added as far as I know. Indeed, some class may not have it.

I think in Python the not-equal comparison is by default the logical not of the equal comparison, when the latter is defined. Here, the only good reason to explicitly add the not-equal comparison is if, in C++, the two operators are not the opposite of one another. I guess this will never be the case for class Frame ?

```python
from __future__ import print_function

class Foo(object):
    def __eq__(self, other):
        print("Foo.__eq__")
        return True
class Bar(object):
    pass

def check(Class):
    a, b = Class(), Class()
    print(a == b)
    print(a != b)

check(Foo)
check(Bar)
```

Reply:
Well, I moved `.def(bp::self == bp::self)` to the visitor, but I did not explicitly expose `.def(bp::self != bp::self)` as it seemed unnecessary

Reply:
It seems `.def(bp::self != bp::self)` _was_ necessary for Python 2. I've pushed it, I hope it fixes everything

Reply:
And `.def(bp::self != bp::self)` reuired the explicit defintion of `!=` in the C++ class too. I took care of it, hope the CI passes this time